src/styles.css

```diff
@@ -115,12 +123,44 @@ a.menu-tab {
   text-decoration: none;
 }
 
-#menu-drawer {
-  height: 100%;
-  overflow: scroll;
+/* WebDesign */
+body {
+  background-color: #e9ddc2;
+}
+
+.buddhanexus {
+  position: absolute;
```

Comment:
this looks like this on my screen:
![image](https://user-images.githubusercontent.com/7022085/74109621-49da5380-4b85-11ea-87a0-abaa656814f1.png)